edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.IoTHub/EdgeAgentConnection.cs

```diff
@@ -331,7 +353,10 @@ enum EventIds
                 UpdatedReportedProperties,
                 ErrorUpdatingReportedProperties,
                 GotModuleClient,
-                GettingModuleClient
+                GettingModuleClient,
+                SendEvent,
+                SendEventClientEmpty,
+                ErrorSendingEvent,
```

Comment:
I'm assuming no retries?

Reply:
I have logic that adds retries, that will be a pr after this one. I wanted to keep my prs small.